src/webapp/components/theme-list/ThemeListTable.tsx

```diff
@@ -32,20 +37,48 @@ export interface ThemeDetail {
     subtitle: string;
     logo: ReactNode;
     palette: string[];
+    users: SharingRule[];
+    userGroups: SharingRule[];
 }
 
 type ThemeListTableProps = Pick<RouteComponentProps, "themes" | "setThemes">;
 
 export default function ThemeListTable({ themes, setThemes }: ThemeListTableProps) {
-    const { compositionRoot } = useAppContext();
+    const { api, compositionRoot } = useAppContext();
     const snackbar = useSnackbar();
     const loading = useLoading();
 
     const [selection, setSelection] = useState<TableSelection[]>([]);
     const [themeEdit, setThemeEdit] = useState<{ type: "edit" | "new"; theme?: Theme }>();
+
+    const [currentUser, setCurrentUser] = useState<User>();
+    getCurrentUser(api).then(user => setCurrentUser(user));
```

Comment:
Components should no have side-effects in the body. If you check the Network tab, you'll see it's requesting non-stop. That should be inside a `React.useEffect`